### PR TITLE
Add new history content type: Interactive Tools

### PR DESCRIPTION
**Pull request by astrovsky01**
Added a new content type to the history to include interactive tools. This change makes it more obvious when an IT is running and accessible using a green pulse: 

https://user-images.githubusercontent.com/40246333/130501939-5fc095f1-a66a-428a-b0b5-c9de6860a80c.mov

It also changes the destination of the view dataset button for these tools to open into a new tab the same way as the other access points for ITs do. 

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

File: client/src/components/History/ContentItem/ContentItem.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ import Placeholder from "./Placeholder";
 import Dataset from "./Dataset";
 import DatasetCollection from "./DatasetCollection";
 import Subdataset from "./Subdataset";
+import InteractiveTool from "./InteractiveTool";
+// import SubDataset from "./SubDataset";
```
Comment (dannon) [inline client/src/components/History/ContentItem/ContentItem.js]:
```suggestion
```

File: client/src/components/History/ContentItem/ContentItem.js [hunk 2]
```diff
@@ -39,6 +41,8 @@ export default {
         Dataset,
         DatasetCollection,
         Subdataset,
+        InteractiveTool,
+        // SubDataset,
```
Comment (dannon) [inline client/src/components/History/ContentItem/ContentItem.js]:
```suggestion
```

File: lib/galaxy/managers/hdas.py [hunk 3]
```diff
@@ -295,6 +296,7 @@ def __init__(self, app: MinimalManagerApp):
 
             'creating_job',
             'rerunnable',
+            # 'job_type',
```
Comment (dannon) [inline lib/galaxy/managers/hdas.py]:
```suggestion
```

File: lib/galaxy/managers/hdas.py [hunk 4]
```diff
@@ -275,6 +275,7 @@ def __init__(self, app: MinimalManagerApp):
             'url',
             'create_time',
             'update_time',
+            'tool_type',
```
Comment (dannon) [inline lib/galaxy/managers/hdas.py]: Check performance, possibly just drop from this view if we can get away with only betawebclient view?

**Comment by dannon** [comment]
@astrovsky01 Can you rebase this work against the latest dev to incorporate it into the history now that it's the default?

**Comment by martenson** [comment]
superseded by #15030

**Pull request closed without merging**